go/types/path.go

```diff
@@ -19,56 +19,23 @@ import (
 
 var annotationRe = regexp.MustCompile("^@([a-z]+)")
 
-type Path []pathPart
+// A Path is an address to a Noms value - and unlike refs (i.e. #abcd...) they can address inlined values.
```

Comment:
Nit: `#abcd...` is a _hash_ in our nomenclature. `Ref` is the object that also includes type information.